### DISCUSSION TITLE
erster erfahrungsbericht

### DISCUSSION THREAD

**Discussion by 3aau** [tbnobody/OpenDTU#1009] [General]
moin,
hier mal meine erfahrungen:
HM1500
esp32
nrf24l01+ (pcb-antenne)

1 panel osten
1 panel westen
2 panel sueden
bester ertrag 8200kwh

opendtu neuste version

distanz dtu zu hm1500 c.a. 5m und eine steinwand

ich sende die daten per mqtt an einen kleinen datenlogger (teensy4.1)
abfrage interval hm1500 alles 5sec.
mqtt    interval        alles 15sec.
der teensy schreibt alle 60sec. auf eine sd-karte und stellt im netz kurven bereit

verwendete daten:
solar/ac/yieldday
solar/ac/power
solar/%s/0/power
solar/%s/1/power
solar/%s/2/power
solar/%s/3/power
solar/%s/4/power

leider werden nicht immer alle daten vom dtu an den teensy gesendet.
ac/power kommt fast immer
die paneldaten sind bei c.a. 30% der datensätze nicht immer alle mit dabei. ebenso der yieldday.
wie oft werde die daten gesendet? gibt es da eine priorisierung?

wenn der hm1500 morgens startet zeigt er offt im MQTT/verbindungsstatus/offline. sendet aber daten an den teensy.
wie oft wird da ein verbindungsaufbau per mqtt versucht? kann ich den hm1500 pingen und/oder aufwecken?

gibt es für den opendtu eine einfache doku über das was auf dem usbport kommt?

ansonsten ein tolles projekt. danke

**Comment by Omega13x** [comment]
Sind die "8200kwh" der Tagesertrag? Wenn ja, musst du ja nicht mehr arbeiten gehen. Sind bei 0,14€/kwh 1148€ am Tag. Guter Schnitt.

**Reply by 3aau** [reply]
sorry, da hab ich was verwechsel. das sind die daten von meinem terassenminifusionskraftwerk.
8200wh ist richtig.

**Comment by 3aau** [comment]
moin,
hier mal meine erfahrungen:
HM1500
esp32
nrf24l01+ (pcb-antenne)

1 panel osten
1 panel westen
2 panel sueden
bester ertrag 8200kwh

opendtu neuste version

distanz dtu zu hm1500 c.a. 5m und eine steinwand

ich sende die daten per mqtt an einen kleinen datenlogger (teensy4.1)
abfrage interval hm1500 alles 5sec.
mqtt interval alles 15sec.
der teensy schreibt alle 60sec. auf eine sd-karte und stellt im netz kurven bereit

verwendete daten:
solar/ac/yieldday
solar/ac/power
solar/%s/1/power
solar/%s/2/power
solar/%s/3/power
solar/%s/4/power

meinen fehler im MQTT client gefunden. nun werden immer alle daten vom tennsy eingefangen.
was ich noch nicht verstanden habe ist der ablauf wenn die DTU die verbindung zum MQTT verliert (zeit überschreitung weil teensy in arbeit ertrinkt, ist diese zeit einstellbar).
bleibt sie dann offline oder versucht sie es nach einer gewissen zeit mit einem neuen connect? (ist diese zeit einstellbar)

 danke
